website_hr_recruitment_legal/templates/apply.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Tecnativa - Jairo Llopis
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<data>
+
+    <template id="apply"
+            inherit_id="website_hr_recruitment.apply"
+            name="Add legal acceptance checkbox">
+        <xpath expr="//*[hasclass('form-group')][//*[hasclass('o_website_form_send')]"
```

Comment:
Typo... you have an extra square bracket... 

```suggestion
        <xpath expr="//*[hasclass('form-group')]//*[hasclass('o_website_form_send')]"
```

Reply:
Thanks! Actually I had one less :grin: